packages/chip/src/Chip.tsx

```diff
@@ -0,0 +1,41 @@
+import { ForwardedRef, forwardRef } from 'react'
+import { ChipProps } from './types'
+import {
+  ChipWrapperStyle,
+  ChipLeftDecoratorStyle,
+  ChipRightDecoratorStyle,
+} from './ChipStyles'
+
+function Chip(props: ChipProps, ref?: ForwardedRef<HTMLInputElement>) {
+  const {
+    children,
+    onClick,
+    variant = 'positive',
+    leftDecorator,
+    rightDecorator,
+    ...rest
+  } = props
+
+  const hasLeftDecorator = !!leftDecorator
+  const hasRightDecorator = !!rightDecorator
```

Comment:
What if decorator is counter? 0 won't work here, is it intended?

Reply:
decorators should be only ReactNode. 
it doesn’t matter now, because after calling Stan, we decided to remove the decorators for this component for the time being